ReactWindows/ReactNative/Chakra/Executor/NativeJavaScriptExecutor.cs

```diff
@@ -104,6 +114,23 @@ public void RunScript(string script, string sourceUrl)
             }
         }
 
+        private static async Task<bool> HasUpToDateSerializedScript(string binFile)
+        {
+            var lastUpdateTime = Windows.ApplicationModel.Package.Current.InstalledDate;
+            var localFolder = ApplicationData.Current.LocalFolder;
+
+            try
+            {
+                var item = await localFolder.GetItemAsync(binFile);
```

Comment:
@rozele it won't if you call `.Result` like I did above, instead will throw that exception.  Weird behavior.